client/src/components/Header/Header.js

```diff
@@ -34,97 +34,83 @@ export default class Header extends React.Component {
-                                            )}
-                                        </div>
-                                        <hr/>
-                                        <li><Button size="sm" variant="link-secondary" style={{padding: 0}}>Log in</Button></li>
-                                        <li><Button size="sm" variant="link-secondary" style={{padding: 0}}>Sign up</Button></li>
-                                    </ul>
-                                </div>
-                            </CheeseburgerMenu>
+                    </div>
+                </div>
 
+                <div className="center">
+                    {/* Desktop and Tablet */}
+                    {window.innerWidth >= 576 && <>
+                        <div className="search-bar-container">
+                            <SearchBar/>
                         </div>
-                    </>
-                }
+                    </>}
+
+                    {/* Mobile */}
+                    {window.innerWidth < 768 && <>
+                        <Modal className="mobile-search-modal" show={this.state.isSearchModalOpen} onHide={() => {this.setState({isSearchModalOpen: false})}} animation={false}>
+                            <Modal.Header closeButton>
+                                <img className='header-icon' src={process.env.PUBLIC_URL + '/MM-logo-rev-3.png'} alt="Monuments and Memorials Logo" width='35px' height='35px'/>
+                            </Modal.Header>
+                            <Modal.Body>
+                                <SearchBar onCloseModal={() => {
+                                    this.setState({isSearchModalOpen: false})
+                                }}/>
+                            </Modal.Body>
+                        </Modal>
+                    </>}
+                </div>
+
+                <div className="right">
+                    {/* Desktop */}
+                    {window.innerWidth >= 992 && <>
+                        <Button size="sm" variant="link-secondary">Log in</Button>
+                        <Button size="sm">Sign up</Button>
+                    </>}
+                    {/* Mobile */}
+                    {window.innerWidth < 768 && <>
+                        <Button variant="link-secondary" className="search-icon" onClick={() => this.setState({isSearchModalOpen: true})}>
+                            <i className="material-icons">
+                                search
+                            </i>
+                        </Button>
+                    </>}
+                    {/* Tablet and Mobile */}
+                    {window.innerWidth < 992 && <>
+                        <Button variant="link-secondary" className="menu-icon" onClick={() => this.setState({isMenuOpen: true})}>
+                            <i className="material-icons">
```

Comment:
FYI @bxsmith007 we have material icon font loaded on the site so you can just do icons this way - much easier than adding images. Check here for info https://material.io/resources/icons/. If you click an icon, you can see on the left side how to add it via the font